### DISCUSSION TITLE
Teammate lost access to auto-saved diagrams on his device via app.diagrams.net

### DISCUSSION THREAD

**Discussion by arvindnarang7** [jgraph/drawio#2389] [Q&A]
Hey Community! 
one of my team-members was using the tool and lost access to auto-saved diagrams from before the weekend. the progress is set to 5 days back or so for some reason. He can see till diagram(19).drawio on his device from about 5 days ago and nothing after that. 

Can someone help out with this ?

**Comment by davidjgraph** [comment]
Where was the diagram being saved to?
